build.gradle

```diff
@@ -672,6 +672,12 @@ project(':streams') {
     testCompile libs.junit
   }
 
+  configurations {
+    // manually exclude unnecessary zkclient transitive dependencies, remove this after KIP-4
+    compile.exclude module: 'jline'
+    compile.exclude module: 'netty'
+  }
+
```

Comment:
The reason I think it should be located by the zkclient dependency is your comment string. It doesn't sound like you are trying to exclude them everywhere (If a dependency was added that needed them you would want them), but just for zkclient. 

Since this section and zkclient need to be removed at the same time I like the idea of co-locating them so cleanup is not missed. 

Really not a huge deal, so I am okay with either. I just wanted to suggest a potential improvement.